salt/console-frontend/templates/PNDA_nginx.conf.tpl

```diff
@@ -20,12 +24,36 @@ server {
 	}
 
 	location /socket.io/ {
-    	proxy_pass http://websocket;
-    	proxy_http_version 1.1;
-    	proxy_set_header Upgrade $http_upgrade;
-    	proxy_set_header Connection "upgrade";
-    }
+    		proxy_http_version 1.1;
+    		proxy_set_header Upgrade $http_upgrade;
+    		proxy_set_header Connection "upgrade";
+    		proxy_pass http://data_manager_api;
+    	}
+
+    	location /api/dl {
```

Comment:
I don't think there's anything running in the browser that requires the data logger API?